AddPipilineRunNotebookLimitaion (#115188)
Azure/azure-content: articles/synapse-analytics/troubleshoot/troubleshoot-synapse-studio.md

---
title: Troubleshoot Synapse Studio
description: Troubleshoot Synapse Studio
author: JeneZhang
ms.service: synapse-analytics 
ms.topic: conceptual
ms.subservice: troubleshooting

> ms.date: 04/15/2020 

Reviewer: ```suggestion
ms.date: 10/01/2023
```

ms.author: jingzh

> ms.reviewer: sngun

Reviewer: ```suggestion
ms.reviewer: sngun, wiassaf
```

This URL pattern is looser than the one shown in “Root Cause” section because it allows for us adding new WebSocket-dependent features to Synapse without any potential connectivity issue in the future. 

> ##  Pipeline run notebook activity issue
> 
> ### Symptom
> 
> Add a notebook which contains more than 128 code cells to a pipeline. Pipeline runs failed.
> 
> ![symptom](media/troubleshooting-synapse-studio/pipeline-run-notebook-activity-issue.png)
> 
> ### Root cause: 
> 
> There is a limitation of running cells(cannot be more than 128) when executing a Synapse notebook activity from a pipeline.
> 
> ### Action: 
> 
> You could merge cells in order to reduce the number of cells below 128.
> 

Reviewer: ```suggestion
## Message queue full or is completed and cannot accept more items

### Symptom

If you add a notebook which contains more than 128 code cells to a pipeline, pipeline runs fail with the error code 6002 and error message: "MessageQueueFullException: The message queue is full or is completed and cannot accept more items." 

:::image type="content" source="media/troubleshooting-synapse-studio/pipeline-run-notebook-activity-issue.png" alt-text="Screenshot from the Azure portal showing error code 6002 in a sample notebook step.":::

### Root cause: 

There is a limitation of 128 of cells when executing a Synapse notebook activity from a pipeline.

### Action: 

You could merge cells in order to reduce the number of cells below 128.

```
